resources/qml/ActionButton.qml

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+import QtQuick.Layouts 1.3
+
+import UM 1.1 as UM
+
+Button
+{
+    id: button
+    property alias cursorShape: mouseArea.cursorShape
+    property var iconSource: ""
+    property var color: UM.Theme.getColor("primary")
+    property var hoverColor: UM.Theme.getColor("primary_hover")
+    property var disabledColor: color
+    property var textColor: UM.Theme.getColor("button_text")
+    property var textHoverColor: UM.Theme.getColor("button_text_hover")
+    property var textDisabledColor: textColor
+    property var textFont: UM.Theme.getFont("action_button")
+    property var cornerRadius: 2 * screenScaleFactor
```

Comment:
This was fixed for some of them but not for all of them. Why is that?

Reply:
I didn't convert all the `var` into `alias` because some variables are used in the same property in a Rectangle. For instance `color`, `hoverColor` and `disabledColor` are three color values for the same rectangle. In the rectangle I define `color: button.enabled ? (button.hovered ? button.hoverColor : button.color) : button.disabledColor`

If is possible to do it with `alias`, I don't know how. That's why I keep it like this.